### PR TITLE
Adds Compose preview to OAuth device grant

### PR DESCRIPTION
**Pull request by kul3r4**
Adds Compose preview to OAuth device grant

### CODE REVIEW

File: WearOAuth/oauth-device-grant/src/main/java/com/example/android/wearable/oauth/devicegrant/AuthDeviceGrantActivity.kt [hunk 1]
```diff
@@ -59,42 +62,69 @@ class AuthDeviceGrantActivity : ComponentActivity() {
...
+    }
+}
+
+@OptIn(ExperimentalHorologistApi::class)
+@Composable
+fun AuthenticateScreen(
+    statusCode: Int,
+    resultMessage: String,
+    context: Context,
+    startAuthFlow: (context: Context) -> Unit
```
Comment (yschimke) [inline WearOAuth/oauth-device-grant/src/main/java/com/example/android/wearable/oauth/devicegrant/AuthDeviceGrantActivity.kt:85]: I'd change this to

```suggestion
    startAuthFlow: () -> Unit
```

File: WearOAuth/oauth-device-grant/src/main/java/com/example/android/wearable/oauth/devicegrant/AuthDeviceGrantActivity.kt [hunk 2]
```diff
@@ -59,42 +62,69 @@ class AuthDeviceGrantActivity : ComponentActivity() {
...
         }
     }
 }
+
+@WearPreviewDevices
+@WearPreviewFontScales
+@Composable
+fun AuthenticateScreenPreview() {
+    AuthenticateScreen(
+        0,
```
Comment (yschimke) [inline WearOAuth/oauth-device-grant/src/main/java/com/example/android/wearable/oauth/devicegrant/AuthDeviceGrantActivity.kt:125]: named params would help here

**Comment by JohnZoellerG** [comment]
Approved!

File: WearOAuth/oauth-device-grant/src/main/java/com/example/android/wearable/oauth/devicegrant/AuthDeviceGrantActivity.kt [hunk 3]
```diff
@@ -59,42 +60,76 @@ class AuthDeviceGrantActivity : ComponentActivity() {
...
     }
 }
+
+@WearPreviewDevices
+@WearPreviewFontScales
+@Composable
+fun AuthenticateScreenPreview() {
+    AuthenticateScreen(
+        statusCode = R.string.status_retrieved,
+        resultMessage = "User name",
```
Comment (yschimke) [inline WearOAuth/oauth-device-grant/src/main/java/com/example/android/wearable/oauth/devicegrant/AuthDeviceGrantActivity.kt:121]: nit: For future, "User name" isn't the most exciting name here in screenshots.

Maybe something with more character like a "Bobby Bonson" or a "Randle McMurphy"?

**Review by JohnZoellerG — COMMENTED** [review]
lgtm

**Pull request merged**